src/Momento.Sdk/Auth/EnvMomentoTokenProvider.cs

```diff
@@ -17,8 +17,11 @@ public class EnvMomentoTokenProvider : ICredentialProvider
     public string ControlEndpoint { get; private set; }
     /// <inheritdoc />
     public string CacheEndpoint { get; private set; }
+
+    #if !BUILD_FOR_UNITY
```

Comment:
I think it's okay to leave this in place regardless of the build target? Just want to try to minimize the number of places we have to keep track of this, especially in the middle of a class/file.

Reply:
Sure, that makes sense. I'll sweep through and remove relevant directives.